sdk/core/core-http/lib/util/constants.ts

```diff
@@ -94,5 +94,21 @@ export const Constants = {
      * @type {string}
      */
     USER_AGENT: "User-Agent"
-  }
+  },
+
+  /**
+   * Marker for atom metadata.
+   *
+   * @const
+   * @type {string}
+   */
+  XML_METADATA_MARKER: "$",
+
+  /**
+   * Marker for atom value.
+   *
+   * @const
+   * @type {string}
+   */
+  XML_VALUE_MARKER: "_"
```

Comment:
The newly added XML_METADATA_MARKER and XML_VALUE_MARKER constants are not being used anywhere. Please remove them

Reply:
Updated